Fix Compilation issue in ,
       - ebtables.c error: 'brnf_call_ebtables' undeclared is defined under CONFIG_BRIDGE_NETFILTER macro protection
         as we are not using BRIDGE_NETFILTER

diff --git a/net/bridge/netfilter/ebtables.c b/net/bridge/netfilter/ebtables.c
--- a/net/bridge/netfilter/ebtables.c
+++ b/net/bridge/netfilter/ebtables.c
@@ -2407,13 +2407,17 @@
 	}
 
 	printk(KERN_INFO "Ebtables v2.0 registered\n");
+#ifdef CONFIG_BRIDGE_NETFILTER
 	brnf_call_ebtables = 1;
+#endif
 	return 0;
 }
 
 static void __exit ebtables_fini(void)
 {
+#ifdef CONFIG_BRIDGE_NETFILTER
 	brnf_call_ebtables = 0;
+#endif
 	nf_unregister_sockopt(&ebt_sockopts);
 	xt_unregister_target(&ebt_standard_target);
 	printk(KERN_INFO "Ebtables v2.0 unregistered\n");
